src/extension/browserAction/previously-enabled.html

```diff
@@ -0,0 +1,26 @@
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="normalize.css">  
+  <link rel="stylesheet" href="styles2FAN.css">
+  <link href="https://fonts.googleapis.com/css?family=Rubik:300,400,700" rel="stylesheet">
+  <script src="../generatedJS/confirmation.bundle.js"></script>
```

Comment:
I think best practice would be for this page to have its own JS file instead of reusing `confirmation.bundle.js` so that if anything changes on this page, we can isolate the changes and not impact other pages. I can def help you do that.